composer.json

```diff
@@ -27,9 +27,9 @@
         "doctrine/doctrine-bundle": "^1.8 || ^2.0",
         "doctrine/orm": "^2.5",
         "doctrine/persistence": "^1.3.4",
-        "sonata-project/admin-bundle": "^3.61",
-        "sonata-project/core-bundle": "^3.14",
+        "sonata-project/admin-bundle": "dev-master as 3.61",
```

Comment:
What is the goal of this ?
Doesn't this say we should work on the AdminBundle 4.0 release before the DoctrineORMBundle release ?

Reply:
I upgrade sandbox to:
- symfony 4.4
- behat 3 (with runnable tests)
- webpack in frontend (working but need more work)

For now I want:
- drop CoreBundle becouse it make so many conflict
- update sandbox: update block or disabled where it is possible  

Then i can release sandbox demo which I need to my work.

Reply:
BTW I already install Admin + ORM : [http://demo.wbloszyk.pl](http://demo.wbloszyk.pl)